CVE-2012-2122/data/docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: '2'
+services:
+ mysql:
+   image: vulhub/mysql:5.5.23
```

Comment:
Could you please download the Docker image, put it to the data folder and install it from file?